File: jupyter2/problems/variables/point_line_distance/jl/point_line_distance.jl
### begin my answer

### end my answer

function main()
    @assert abs(point_line_distance(-3.0, 1.0, 1.0, -2.0, 4.0, -3.0) - 1.0) < 1.0e-5
    @assert abs(point_line_distance( 2.0, 2.0, 4.0,  0.0, 1.0,  0.0) - 2.12132) < 1.0e-5
    @assert abs(point_line_distance( 1.0, 1.0, 4.0,  3.0, 3.0, -2.0) - 3.60555) < 1.0e-5
    println("OK")
end

main()

File: jupyter2/problems/variables/point_line_distance/rs/point_line_distance.rs
/** begin my answer */

/** end my answer */

fn main() {
    assert!((point_line_distance(-3.0, 1.0, 1.0, -2.0, 4.0, -3.0) - 1.0).abs() < 1.0e-5);
    assert!((point_line_distance( 2.0, 2.0, 4.0,  0.0, 1.0,  0.0) - 2.12132).abs() < 1.0e-5);
    assert!((point_line_distance( 1.0, 1.0, 4.0,  3.0, 3.0, -2.0) - 3.60555).abs() < 1.0e-5);
    println!("OK")
}
